telemetryui/migrations/versions/a78598efefae_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
+
+Revision ID: a78598efefae
+Revises: 466cf2f35d67
+Create Date: 2017-10-16 04:39:31.861350
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a78598efefae'
+down_revision = '466cf2f35d67'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('record_attachments',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('record_id', sa.Integer(), nullable=True),
+    sa.Column('file_path', sa.String(), nullable=False),
+    sa.Column('mime_type', sa.String(), nullable=True),
+    sa.ForeignKeyConstraint(['record_id'], ['records.id'], ondelete='SET NULL'),
```

Comment:
Should a new migration be added instead revising this line in the old migration?

Reply:
Another migration is not necessary since there is no change on rows, in fact I did a ```flask migrate``` and I got a message saying that there was no change on table.  Version a78598efefae replaces 1b5110b63a63. 

If this set of commits are merged there will be only one additional version a78598efefae for upgrade, 1b5110b63a63 will not exists once all the commits in the set are accepted, because I was careful to downgrade from 1b5110b63a63 to 466cf2f35d67 before generating the migration to a78598efefae. You can see that [here](https://github.com/clearlinux/telemetrics-backend/pull/26/files/5211fd21cbf897bb9032a1c98dade31b161bc2a2#diff-ca6171a6ebd7e979677d2ec0051017fd) where 

```
Revision ID: a78598efefae
Revises: 466cf2f35d67
```
a78598efefae is not only a change on the constraint is a whole new table.

Reply:
Okay, yeah, I should have clarified that I was concerned about revising the new migration in this patch series. It sounds like you want to squash the two new migrations into one, and I would be okay with that.